components/product/ProductShelf.tsx

```diff
@@ -15,20 +15,28 @@ import { mapProductToAnalyticsItem } from "apps/commerce/utils/productToAnalytic
 export interface Props {
   products: Product[] | null;
   title?: string;
-  description?: string;
   layout?: {
     headerAlignment?: "center" | "left";
     headerfontSize?: "Normal" | "Large";
   };
+  /**
+  * @description Choose between the carousel types (Normal with arrows or the last item to be a link to more products)
+  */
+  /** @default "Arrows" */
+  carouselType?: "Arrows" | "See More Link";
+  seeMoreText?: string;
+  seeMoreLink?: string;
```

Comment:
Quando voce tiver um contexto de algo que abrange mais de uma variavel, prefira criar um objeto
```suggestion
  seeMore?: { text: string; link: string }
```